examples/frameworks/strands_demo/bedrock_agentcore/scripts/run_nat_no_OTEL.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+if [[ -z "$NAT_CONFIG_FILE" ]]; then
+  echo "NAT_CONFIG_FILE not set" >&2
+  exit 1
+fi
+
+export NVIDIA_API_KEY=$(aws secretsmanager get-secret-value --secret-id 'nvidia-api-credentials' \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)